build/pkg/build.sh

```diff
@@ -88,7 +88,7 @@ clone_source(){
 build(){
     pushd $TMPDIR/$BUILDDIR/pkg/src > /dev/null || logerr "Cannot change to src dir"
     find . -depth -name \*.mo -exec touch {} \;
-    touch `find gui/help -depth -name \*.in | sed -e 's/\.in$//'`
+    find gui/help -depth -name \*.in | sed -e 's/\.in$//' | xargs touch
```

Comment:
this will run into trouble if there are files spaces in the name ... but so would the old version have ... so I guess that is fine :)